### ISSUE TITLE
`tasks/claimFunds.js` doesn't print any output

### ISSUE THREAD

**Issue by reverendus** [gnosis/dx-services#11]
It's a bit inconvenient that running `tasks/claimFunds.js` doesn't produce any output, regardless of whether something actually got claimed or not. It always exits silently:

The way I run `tasks/claimFunds.js` is:
```
docker run \
  -e MNEMONIC="$MNEMONIC" \
  -e ETHEREUM_RPC_URL=https://rinkeby.infura.io \
  -e NODE_ENV=pre \
  -e NETWORK=rinkeby \
  --mount type=bind,source=$(pwd)/conf,destination=/usr/src/app/custom_conf \
  -e CONFIG_FILE=/usr/src/app/custom_conf/bots.js \
  $@ \
  gnosispm/dx-services:develop \
  node src/tasks/claimFunds.js
```

**Comment by anxolin** [comment]
It should be fixed in version `0.6.8`, please try again and let us know!

**Comment by reverendus** [comment]
Fixed. Closing.
